parastell.py

```diff
@@ -558,44 +489,33 @@ def expand_ang(ang_list, num_ang):
     
     Returns:
         ang_list_exp (list of float): interpolated list of angles (deg).
-        num_ang_list (list of int): number of angles interpolated between
-            user-supplied list.
     """
     # Initialize interpolated list of angles
     ang_list_exp = []
-    # Initialize list of number of angles interpolated between user-supplied
-    # list
-    num_ang_list = []
 
     # Compute total angular extent of supplied list
     ang_ext = ang_list[-1] - ang_list[0]
     
     # Loop over supplied angles
-    for i, ang in enumerate(ang_list[:-1]):
+    for ang, next_ang in zip(ang_list[:-1], ang_list[1:]):
         # Compute difference between current and next angles
-        ang_diff = ang_list[i + 1] - ang
+        ang_diff = next_ang - ang
         # Compute ratio of angular difference and total angular extent
         ratio = ang_diff/ang_ext
         
         # Compute number of angles to interpolate
         n_ang = round(ratio*num_ang)
```

Comment:
There may be a clearer way to do this by calculating a `max_ang_diff = ang_ext/num_ang` outside of the loop, and then calculating `n_ang = ceil( (next_ang - ang) / max_ang_diff )`